kolibri/plugins/style_guide/assets/src/app.js

```diff
@@ -3,6 +3,9 @@ const Vue = require('kolibri.lib.vue');
 const RootVue = require('./views');
 const router = require('kolibri.coreVue.router');
 const { navigationMenuRoutes } = require('./views/shell/navigation-menu');
+const Vuep = require('./third_party/vuep/vuep');
```

Comment:
Thanks for the review and suggestion! I've removed the local copy, and added the dependencies via Yarn. Please take another look.